diff --git a/install_1.sh b/install_1.sh
index 14ea60a..32bdf87 100755
--- a/install_1.sh
+++ b/install_1.sh
@@ -4,7 +4,7 @@ sudo add-apt-repository ppa:git-core/ppa
 sudo add-apt-repository ppa:ubuntu-support-team/gdb
 sudo apt update
 sudo apt install -y wget git gdb
-sudo apt-get install -y zsh curl uzip npm
+sudo apt-get install -y zsh curl unzip npm
 
 git clone https://github.com/gggga0212/vimsetting ~/.config/nvim
 cd ~/.config/nvim
